tools/generate_pv_yaml_rest.sh

```diff
@@ -0,0 +1,393 @@
+                -r|--guihost <Route host name used to route traffic to the spectrum scale GUI service.>
+                [-p|--pvname <name for pv>]
+                [-c|--storageclass <StorageClass for pv>]
+                [-a|--accessmode <AccessMode for pv>]
+                [-h|--help]
+
+
+Example 1: Single Fileystem
+	In this setup there is only one fileystem 'gpfs0' and directory from the same fileystem is being used as volume.
+
+	$0 --filesystem gpfs0 --linkpath /ibm/gpfs0/fileset1/.volumes/staticpv --size 10 --pvname mystaticpv --guihost ibm-spectrum-scale-gui-ibm-spectrum-scale.apps.hci-cluster.cp.fyre.ibm.com
+
+
+Example 2: Two or More Filesystem
+	In this setup there are two filesystems 'gpfs0' and 'gpfs1'. gpfs0 is configured as primary fileystem in Spectrum-scale-csi setup. User want to create volume from the directory present in the gpfs1 filesystem. Say the directory in the gpfs1 is /ibm/gpfs1/dir1. As a first step user will create softlink  /ibm/gpfs1/dir1 --> /ibm/gpfs0/fileset1/.volumes/staticpv1 and then run following command to generate the pv.yaml.
+
+	$0 --filesystem gpfs1 --linkpath /ibm/gpfs0/fileset1/.volumes/staticpv1 --size 10 --pvname mystaticpv1 --guihost ibm-spectrum-scale-gui-ibm-spectrum-scale.apps.hci-cluster.cp.fyre.ibm.com
+
+Example 3: Fileset based volume
+	This example shows how to create a volume from a fileset 'fileset1' within the filesyetem 'gpfs0'.
+
+	$0 --filesystem gpfs0 --fileset fileset1 --size 10 --pvname mystaticpv --guihost ibm-spectrum-scale-gui-ibm-spectrum-scale.apps.hci-cluster.cp.fyre.ibm.com
+
+	Note: This script does not validate if softlinks are correctly created.
+	      The Path specified for option --linkpath must be valid gpfs path from primary fileystem." 1>&2
+  exit 1
+}
+
+# Generate Yaml
+generate_pv_yaml() {
+  volhandle=$1
+  volname=$2
+  volsize=$3
+  accessmode=$4
+  if [[ -f "${volname}.yaml" ]]; then
+    echo "ERROR: File ${volname}.yaml already exist"
+    exit 2
+  fi
+
+  cat >"${volname}".yaml <<EOL
+# -- ${volname}.yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+        name: ${volname}
+spec:
+  capacity:
+    storage: ${volsize}Gi
+  accessModes:
+    - ${accessmode}
+  claimRef:
+    name: pvc-${volname}
+    namespace: ibm-spectrum-scale-csi
+  csi:
+    driver: spectrumscale.csi.ibm.com
+    volumeHandle: ${volhandle}
+  ${STORAGECLASS}
+EOL
+  echo "INFO: volumeHandle: ${volhandle}"
+  echo "INFO: Successfully created ${volname}.yaml"
+}
+
+# Generate PVC manifest
+generate_pvc_yaml() {
+  volname=$1
+  volsize=$2
+  accessmode=$3
+  if [[ -f "pvc-${volname}.yaml" ]]; then
+    echo "ERROR: File pvc-${volname}.yaml already exist"
+    exit 2
+  fi
+
+  cat >pvc-"${volname}".yaml <<EOL
+# -- pvc-${volname}.yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc-${volname}
+  namespace: ibm-spectrum-scale-csi
+spec:
+  accessModes:
+    - ${accessmode}
+  resources:
+    requests:
+      storage: ${volsize}Gi
+EOL
```

Comment:
${STORAGECLASS} is missing in PVC yaml file

Reply:
````
# oc get pv mystaticpv1
NAME          CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                                    STORAGECLASS   REASON   AGE
mystaticpv1   10Gi       RWO            Retain           Bound    ibm-spectrum-scale-csi/pvc-mystaticpv1   testersc                12m
```
```
# oc get pvc pvc-mystaticpv1
NAME              STATUS   VOLUME        CAPACITY   ACCESS MODES   STORAGECLASS   AGE
pvc-mystaticpv1   Bound    mystaticpv1   10Gi       RWO            testersc       12m
```